modules/addStaticOnLocalStorage.js

```diff
@@ -0,0 +1,117 @@
+import { usersSettings } from './usersSettings.js'
+
+// Добавляет статистику по загрузкам в localStorage
+function addStaticOnLocalStorage() {
+    let elem = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload")
+
+    let promise = new Promise((resolve, reject) => {
+        let observer = new MutationObserver((mutationRecords) => {
+            if (mutationRecords.length > 1) {
+                let saveScanStat = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload.b-popup--edit.js-popup--edit > form > div.b-popup-button.js-upload-submit"); // кнопка по которой срабатывает скрипт
+                let q = document.querySelector("#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div").childElementCount  //Кол-во элементов для фикла
+                let sostav = ' состав неопределен'
+                let docId = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload.b-popup--edit.js-popup--edit > form > input[type=hidden]:nth-child(4)").value
+
+                //it's selector for add new document without the card
+                //document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload > form > input[type=hidden]:nth-child(4)").value
+                let a40 = document.querySelector("#b-case-header > ul.crumb.g-ec > li > span").textContent.split(" ")[20]
+
+
+                for (let i = 1; i < q; i++) {
+                    if (document.querySelector(`#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div > div:nth-child(${i}) > div.r-col > h2 > a > span`) !== null) {
+                        let checker = document.querySelector(`#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div > div:nth-child(${i}) > div.r-col > h2 > a `)
+                            .textContent //полуучает текст
+                            .trim() //Удаляет пробелы "с краёв"
+                            .split(` `) //Правращает в масив по разделителю " "
+                        let stringValueCounter = 0
+                        if (stringValueCounter < 2) {
```

Comment:
and what if `stringValueCounter` is equal to 2 ?
I recommend you to change if-else implementation here, it's confusing and long